buku.py

```diff
@@ -557,7 +567,7 @@ def append_tag_at_index(self, index, tags_in):
         '''
 
         if index == 0:
-            resp = input('Append specified tags to ALL bookmarks? (y/n): ')
+            resp = read_in('Append specified tags to ALL bookmarks? (y/n): ')
```

Comment:
`resp = read_in('Append specified tags to ALL bookmarks? (y/n): ')`

please change it to:

`resp = read_in('Append the tags to ALL bookmarks? (y/n): ')`